documentation/book/con-configure-operator.adoc

```diff
@@ -0,0 +1,112 @@
+[id='configure-cluster-operator-{context}']
```

Comment:
I don't think a concept file should have more than 1 headings. I might be wrong and need to look this up.